pkg/api/frontendsettings.go

```diff
@@ -160,18 +161,7 @@ func (hs *HTTPServer) getFrontendSettings(c *contextmodel.ReqContext) (*dtos.Fro
 		SharedWithMeFolderUID:               folder.SharedWithMeFolderUID,
 
 		Auth: dtos.FrontendSettingsAuthDTO{
-			OAuthSkipOrgRoleUpdateSync:  hs.Cfg.OAuthSkipOrgRoleUpdateSync,
```

Comment:
deprecated or removing the settings? If we are deprecating we should add them back here

Reply:
The initialization has been moved to  line 263: https://github.com/grafana/grafana/pull/79443/files/bfe883b5dd1c4b79aa864842157eb26a20de9a32#diff-cea61b688b82f66d0b664d7a06957747172c3554730a80da1a8f7a13ed9b0f1aR263

I'm gonna remove the init from line 163, because it's misleading.